distribution/carbon-home/bin/dropins.bat

```diff
@@ -0,0 +1,78 @@
+@echo off
```

Comment:
This file should be included in the Kernel feature and not in the distribution of the source.